Create 2020-01-08-paperMetaproteomics.software.md (#398)
usegalaxy-eu/website: _posts/2020-01-05-laborjournal.md

---
site: freiburg
title: "Laborjournal article about Galaxy"
tags: [paper]
supporters:
 - denbi
 - elixir
---

> The **Laborjournal** is a German journal for scientists for complex methods and news written in an easy understandable language. Björn from our team was interviewed about [Galaxy in life science](http://www.laborjournal-archiv.de/epaper/LJ_19_12/50/index.html). It is in German - you can practice your German skills ;-)

Reviewer: ```suggestion
The **Laborjournal** is a German journal for scientists for complex methods and news written in easily understandable language. Björn from our team was interviewed about [Galaxy in life science](http://www.laborjournal-archiv.de/epaper/LJ_19_12/50/index.html). The article is in German - you can practice your German skills ;-)
```
